line_follower_node:
  ros__parameters:
    MAX_SPEED: 6.28
    TURN_LEFT_SPEED: 0.4
    TURN_RIGHT_SPEED: 0.8
    SENSOR_THRESHOLD: 600.0
